DNN Platform/Library/Security/PortalSecurity.cs

```diff
@@ -616,8 +616,6 @@ public string Remove(string inputString, ConfigType configType, string configSou
 
         public void SignIn(UserInfo user, bool createPersistentCookie)
         {
-            InvalidateAspNetSession(HttpContext.Current);
-
```

Comment:
I don't remember the exact detail, but there was a reason for adding this line of code when I made that change. Before I added it, there were cases which allowed reusing the session after logging out and this line prevented that from happening. Therefore, without this line, there could be some security issues. More testing needed.